### PR TITLE
Ratio of dsr

### PR DESCRIPTION
**Pull request by LindaNab**
The main addition is the revised calculation of the DSR (direct standardised rates) + variance and, the calculation of the SRR (standardised rate ratios) + confidence interval construction of the SRR. In addition, functionality is added to plot the SRRs.
Notes on the DSR, SRR + associated variance estimation is [here](https://docs.google.com/document/d/1Slo6FxC2Jv2qrqz5T4rnH_VXJhRajq7bKc_pcuveV5s/edit?usp=sharing).

### CODE REVIEW

**Comment by LindaNab** [comment]
Document that should be visible for the DataLab @andrewscolm: https://docs.google.com/document/d/1Slo6FxC2Jv2qrqz5T4rnH_VXJhRajq7bKc_pcuveV5s/edit?usp=sharing

File: analysis/utils/process_rates.R [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+## Load function fct_case_when() to be used to change level of factors
+source(here("analysis", "utils", "fct_case_when.R")) 
+## Load json file listing demographics and comorbidities
+config <- fromJSON(here("analysis", "config.json"))
+
+# Load rates ---
+## Create vector containing the demographics and comorbidities
+subgroups_vctr <- c(config$demographics, config$comorbidities)
+# Import the standardised mortality rates:
+subgroups_vctr <- c(config$demographics, config$comorbidities)
```
Comment (andrewscolm) [inline analysis/utils/process_rates.R:26]: minor point but this is repeated above.

**Review by andrewscolm — APPROVED** [review]
Looks great

File: analysis/utils/process_rates.R (hunk 1 above)
Comment (LindaNab) [reply analysis/utils/process_rates.R:26]: Good one :)

**Pull request merged**